Description: Add feature custom_inner_attributes to src/lib.rs to fix FTBFS
Author: Peter Michael Green <plugwash@debian.org>
Bug-Debian: https://bugs.debian.org/952228

Index: rust-core-arch-0.1.5/src/lib.rs
===================================================================
--- rust-core-arch-0.1.5.orig/src/lib.rs
+++ rust-core-arch-0.1.5/src/lib.rs
@@ -32,7 +32,8 @@
     wasm_target_feature,
     abi_unadjusted,
     adx_target_feature,
-    external_doc
+    external_doc,
+    custom_inner_attributes
 )]
 #![cfg_attr(test, feature(test, abi_vectorcall, untagged_unions))]
 #![deny(clippy::missing_inline_in_public_items)]
